modules/web/themes/helium/com.haulmont.sample.petclinic/helium-ext-variant-dark-purple-menu.scss

```diff
@@ -0,0 +1,77 @@
+/* Define your theme modifications inside next mixin */
+@mixin com_haulmont_sample_petclinic-helium-ext {
+    --hell-lila: #867AEB;
+    --white: #FFFFFF;
+
+      /* Basic */
+    --primary-color: #6C5CE7;
+    --primary-color_rgb: 108, 92, 231;
+    --secondary-color: #FDD58D;
+    --secondary-color_rgb: 253, 213, 141;
+    --success-color: #83C93C;
+
+      /* Common */
+    --primary-color-shade-1: #4C3CE2;
+    --primary-color-shade-2: #3421DE;
+    --primary-dim-color: #958CEE;
+    --secondary-color-shade-1: #FCC464;
+    --secondary-color-shade-2: #FBB53C;
+    --secondary-contrast-text-color: #2C3436;
+    --secondary-dim-color: #FDDFAA;
+    --success-color-shade-1: #75B332;
+    --success-color-shade-2: #689F2D;
+    --success-dim-color: #A7D874;
+    --text-dim-color: #6B7172;
```

Comment:
I don't recommend changing
```
--text-dim-color
--text-secondary-color
```
because now they have different tone then the other 'gray' colors in the helium theme.

Reply:
Not sure if I accidentally did it. Will double-check. Is this way of configuring the dark theme the "correct" one? I mean it work, but I just made it up on my own, so I was curios if it actually is as intended.

Reply:
done 👍